newswires/client/src/App.tsx

```diff
@@ -222,12 +238,8 @@ export function App() {
 												css={css`
 													padding-bottom: 3px;
 
-													${largeMaxBreakpoint} {
-														margin-right: 8px;
```

Comment:
what's the reason for dropping the margin here? looks like we could do with some gap between the 'under construction' badge and the search input here..?

<img width="891" alt="image" src="https://github.com/user-attachments/assets/4a9b4531-9ac1-4800-8402-014a6b16187b" />